Source/SSH.js

```diff
@@ -138,6 +139,8 @@ export default class SSH {
             ))
           } else reject(err)
         })
+      }).then(function(){
+        SFTP.end();
```

Comment:
Wouldn't this line break `putMulti` because it'll close the sftp on first attempt and it'll be closed for the later?
A good solution would be to only close this if we opened it (instead of receiving it)